src/harness/testcases/WINNF_FT_S_IPR_testcase.py

```diff
@@ -210,37 +211,40 @@ def test_WINNF_FT_S_IPR_1(self, config_filename):
              cbsd.getRegistrationRequest()['cbsdSerialNumber']))
 
     # Get SAS UUT FAD and Test Harness FADs.
-    sas_uut_fad = getFullActivityDumpSasUut(self._sas, self._sas_admin)
+    sas_uut_fad = None
     test_harness_fads = []
-    for test_harness in test_harnesses:
-      test_harness_fads.append(
-          getFullActivityDumpSasTestHarness(
-              test_harness.getSasTestHarnessInterface()))
+    if num_peer_sases:
+        sas_uut_fad = getFullActivityDumpSasUut(self._sas, self._sas_admin)
+        for test_harness in test_harnesses:
+          test_harness_fads.append(
+              getFullActivityDumpSasTestHarness(
+                  test_harness.getSasTestHarnessInterface()))
 
     # Trigger CPAS.
     self.TriggerDailyActivitiesImmediatelyAndWaitUntilComplete()
 
-    # Initialize DPA objects and calculate movelist for each DPA.
-    dpas = []
-    for dpa_config in config['dpas']:
-      dpa = dpa_mgr.BuildDpa(dpa_config['dpaId'], dpa_config['points_builder'])
-      freq_range_low = dpa_config['frequencyRange']['lowFrequency'] / ONE_MHZ
-      freq_range_high = dpa_config['frequencyRange']['highFrequency'] / ONE_MHZ
-      dpa.ResetFreqRange([(freq_range_low, freq_range_high)])
-      dpa.SetGrantsFromFad(sas_uut_fad, test_harness_fads)
-      dpa.ComputeMoveLists()
-      dpas.append(dpa)
     # Heartbeat SAS UUT grants.
     n2_domain_proxy.heartbeatForAllActiveGrants()
     n3_domain_proxy.heartbeatForAllActiveGrants()
     # Get CbsdGrantInfo list of SAS UUT grants that are in an authorized state.
     grant_info = data.getAuthorizedGrantsFromDomainProxies([n2_domain_proxy, n3_domain_proxy])
 
-    # Check grants do not exceed each DPAs interference threshold.
-    for dpa, dpa_config in zip(dpas, config['dpas']):
+    # Initialize DPA objects and calculate movelist for each DPA.
+    dpas = []
```

Comment:
Please can someone else help reviewing these DPA related addition ?

Reply:
In #615 , the DPA code has been updated to support None FAD objects. In these case, the checkInterference will check the SAS UUT active grants generated intereference against the absolute threshold.